### ISSUE TITLE
Enhancement: Export screenshot even if test run in timeout

### ISSUE THREAD

**Issue by kkrieger85** [DanielRuf/website-checks#42]
Current behavior:

Script shows error message lik '[error] SSLLabs TimeoutError: waiting for selector ".reportTime" failed: timeout 120000ms exceeded'

Proposed behavior:

Script exports screenshot from current state of websites test. So user could see, if test is really still running or something else happend

**Comment by DanielRuf** [comment]
Right, SSLLabs is one of the slowest and takes a very long time to finish the reports.

So something like `ssllabs-failed.pdf` as fallback in the error handlers?

**Comment by kkrieger85** [comment]
Yes. Fallback sounds good. 
Issue in my latest test results in #43 
So with a fallback pdf I had less problems to identify an issue wich is not a missing result ;)

**Comment by MyIsaak** [comment]
I included a PR for this issue.
